### ISSUE TITLE
[doc] WASI-tutorials etc. not published

### ISSUE THREAD

**Issue by jeschkies** [bytecodealliance/wasmtime#2536]
Forgive me if I'm mistaken. It seems the [WASI-tutorial](https://github.com/bytecodealliance/wasmtime/blob/main/docs/WASI-tutorial.md) and related WASI docs are not published to https://docs.wasmtime.dev/. At least I couldn't find them through the page but did through Google.

**Comment by alexcrichton** [comment]
Yes at this time these aren't published (they're inherited from original documentation some time ago), but we could perhaps clean them up and publish them too though!
